### DISCUSSION TITLE
Have you tried memex? it is open sourced raindrop.io alternative

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#6] [Q&A] [answered]
Fabulous and thanks! A quick comment on your current state-of-art workflow: have you tried [memex](https://getmemex.com/)? it is open sourced raindrop.io alternative and thought you might like that :p

_Originally posted by @liutiming in https://github.com/BryanJenksCommunity/FAQ/issues/5#issuecomment-727948145_

**Comment by tallguyjenks** [comment]
@liutiming I have not, and given that its not entirely free and i like the raindrop interface better I'm unlikely to migrate, but i will check it out :)

**Comment by tallguyjenks** [comment]
OK Woah. Looking at some of their features and the main cost being just automatic sync? this actually warrants a good thorough look, Thank you for the suggestion!

**Comment by tallguyjenks** [comment]
- the lack of free mobile syncing is annoying though i don't use that raindrop feature too much.
- the notes per bookmark is a plus. 
- given the open-sourcing of the project likely there's a way or will be a way of easily extracting such data from the bookmarks. 
- Local storage is a massive plus
- Interface is very nice

I'll have to think about this but this is definitely looking like a possibility. If monetary contribution supports open source then it could be a 2 bird one stone option, @liutiming Excellent suggestion thank you!

**Comment by tallguyjenks** [comment]
OMG and the template posting! https://www.notion.so/Text-Export-Templates-bb1794c15dea4e588a59b52d37ca0070

this would make it much easier to import meta data into new inputs in my Obsidian vault! That settles it, migration will occur

**Reply by mnott** [reply]
OMG!

I've tried out the templating. Here's a very simple shot that works perfectly well with Obsidian:

```
[{{{PageTitle}}}]({{{PageUrl}}})

{{#Notes}}

> "{{{NoteHighlight}}}"
- {{{NoteText}}}

{{/Notes}}


```

That's amazing. I'm now just editing / highlighting on any page, and later just hit that export button and drop it into a note in Obsidian.

**Comment by liutiming** [comment]
fab. will be excited to see what is your new workflow like :)

**Comment by tallguyjenks** [comment]
@liutiming oh there will be a video on this 🤣

**Comment by tallguyjenks** [comment, marked as answer]
Converted from issue

**Comment by montemartin** [comment]
I'm curious if this ever happened? 

I've been following Memex for a while and want to love it. They have some excellent features. Integrating highlighting/notes, local storage first, and their business model is unique and quite appealing (an org with all the benefits of an indie).

But Raindrop's UI is so polished and easy to use that I have a hard time getting onboard with the Memex. Would love to hear further thoughts on how the two compare and especially how to integrate with Obsidian.

**Reply by tallguyjenks** [reply]
ended up going with raindrop because it works so well for my use case. with memex i was really only using it for timestamped youtube notes but now the media extended plugin in obsidian can take care of that
